<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaneControlsWidget</class>
 <widget class="QWidget" name="PlaneControlsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Control surfaces</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Fan rpm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="spinFanRPM">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Elevator left</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spinElevatorL_angle">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Elevator right</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="spinElevatorR_angle">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Alerone left</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="spinAleroneL_angle">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Alerone right</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="spinAleroneR_angle">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Rudder</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="spinRudder_angle">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Flight controls</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Velocity</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="spinVelocity">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Pitch</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spinPitch">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Roll</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="spinRoll">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Yaw</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="spinYaw">
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Altitude</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="spinAltitude">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Latitude</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="spinLatitude">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="spinLongitude">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinFanRPM</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlaneControlsWidget</receiver>
   <slot>onValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinElevatorL_angle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlaneControlsWidget</receiver>
   <slot>onValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinElevatorR_angle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlaneControlsWidget</receiver>
   <slot>onValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinAleroneL_angle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlaneControlsWidget</receiver>
   <slot>onValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinAleroneR_angle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlaneControlsWidget</receiver>
   <slot>onValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinRudder_angle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlaneControlsWidget</receiver>
   <slot>onValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinVelocity</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlaneControlsWidget</receiver>
   <slot>onValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinPitch</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlaneControlsWidget</receiver>
   <slot>onValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinRoll</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlaneControlsWidget</receiver>
   <slot>onValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>607</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>668</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinYaw</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlaneControlsWidget</receiver>
   <slot>onValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinAltitude</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlaneControlsWidget</receiver>
   <slot>onValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinLatitude</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlaneControlsWidget</receiver>
   <slot>onValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>664</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinLongitude</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PlaneControlsWidget</receiver>
   <slot>onValueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>662</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onValueChange()</slot>
 </slots>
</ui>
